.registry/server.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://static.modelcontextprotocol.io/schemas/2025-10-17/server.schema.json",
+  "name": "io.github.googleapis/genai-toolbox",
+  "description": "MCP Toolbox for Databases enables your agent to connect to your database.",
+  "status": "active",
+  "repository": {
+    "url": "https://github.com/googleapis/genai-toolbox",
+    "source": "github"
+  },
+  "version": "0.18.0",
+  "packages": [
+    {
+      "registryType": "oci",
+      "registryBaseUrl": "https://artifactregistry.googleapis.com",
+      "identifier": "us-central1-docker.pkg.dev/database-toolbox/toolbox/toolbox:0.18.0",
+      "transport": {
+        "type": "streamable-http",
+        "url": "http://{host}:{port}"
+      },
+      "packageArguments": [
+        {
+            "type": "named",
+            "name": "--tools-file",
```

Comment:
If we are using the container I would assume we may need to add "runtimeArguments": to mount the tools.yaml into the container (https://github.com/modelcontextprotocol/registry/blob/main/docs/reference/server-json/generic-server-json.md#filesystem-server-with-multiple-packages)